ios/engine/KMEI/KeymanEngine/Classes/Resource Management/PackageInstallViewController.swift

```diff
@@ -349,8 +349,6 @@ public class PackageInstallViewController<Resource: LanguageResource>: UIViewCon
     Manager.shared.shouldReloadKeyboard = true
     self.pickingCompletionHandler(selectedResources.map { $0.typedFullID })
 
-    self.completionHandler(selectedResources.map { $0.typedFullID })
```

Comment:
`completionHandler` was renamed to `pickingCompletionHandler` in a very recent PR.

Reply:
I see it was accidentally left in from #4338 
https://github.com/keymanapp/keyman/pull/4338/files#diff-ffe9df0bee7c059c5847b53a7f442ab3d4478d04b1d036139df0638a02949bbaR352